### PR TITLE
initial content replacements, structural edits to follow soon

### PR DESCRIPTION
**Pull request by urcades**

### CODE REVIEW

File: content/understanding-urbit/urbitos1-osn.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
++++
+title = "Urbit OS1 –> OSN"
```
Comment (matildepark) [inline content/understanding-urbit/urbitos1-osn.md]: OS¹ -> OSⁿ? Hehe.

Comment (urcades) [reply content/understanding-urbit/urbitos1-osn.md]: Yeah, I'd personally prefer this styling tbh

Comment (matildepark) [reply content/understanding-urbit/urbitos1-osn.md]: Up to you

File: content/understanding-urbit/urbit-id.md [hunk 2]
```diff
@@ -9,27 +9,27 @@ hide_previous_title = "true"
...
 
 First, let’s talk about Urbit ID’s implementation and design goals, then about where we envision it going in the future.
 
-If you’re curious to see exactly how this system works, the code is of course [open source](). Including the source for our [phonemic naming system]() and the [visual representations](). There’s even a [Figma plugin]() for using them. There’s also an interface for interacting with Urbit ID whose source is [open too]().
+If you’re curious to see exactly how this system works, the code is of course [open source](https://github.com/urbit/urbit). Including the source for our [phonemic naming system]() and the [visual representations](https://github.com/urbit/sigil-js). There’s even a [Figma plugin](https://github.com/urbit/sigil-figma-plugin) for using them. There’s also an interface for interacting with Urbit ID whose source is [open too](https://github.com/urbit/bridge).
```
Comment (urcades) [inline content/understanding-urbit/urbit-id.md]: The link for the source for the "phonemic naming system" needs to be inserted here, I have no idea what this is referring to.

**Review by urcades — COMMENTED** [review]
Howdy @matildepark + @eriknewton + @galenwp , please give this update a general run-down.

The main thing to make sure is correct (i think) is the linking, since links were not provided in the source docs.

File: content/understanding-urbit/urbit-id.md (hunk 2 above)
Comment (matildepark) [reply content/understanding-urbit/urbit-id.md]: Could be [urbit-ob](https://github.com/urbit/urbit-ob/blob/master/src/internal/co.js). Or the appropriate section from hoon.hoon.

File: content/blog/simple-durable-yours.md [hunk 3]
```diff
@@ -1,11 +1,8 @@
 +++
 title = "Simple, Durable, Yours"
+date = 2019-10-17
 description = "The core ideas that Urbit is built around"
-weight = 1
 [extra]
```
Comment (matildepark) [inline content/blog/simple-durable-yours.md:5]: I feel like these could use a touch up for their description and the author metadata. They look out of place in the blog directory.

image

File: content/understanding-urbit/history-people.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+Tlon has been around since 2013 or so, and is primarily focussed on the development of Urbit OS and its user experience. Urbit.live was started in 2018 to make the onramp into Urbit as easy as possible. Both companies contribute regularly to the code base.
+
+Tlon was originally granted about half of the Urbit ID address space, and has used that to fund its operations so far. Over time, Tlon plans to offer hosting and services to make using Urbit as easy as possible. 
+
+Tlon is also working to set up an urbit.org foundation. This is still in process, and is expected to happen in 2020.
+
+Urbit.live started selling Urbit IDs very early once that became possible, and also plans to offer services within the Urbit ecosystem. 
+
+The community of contributors and core developers has been steadily contributing to the code base since before Tlon existed, and they’re still going strong. They can primarily be found on Urbit itself, in `~dopzod/urbit-help`, the [urbit-dev mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) and on [GitHub](https://github.com/urbit/urbit/issues).
```
Comment (matildepark) [inline content/understanding-urbit/history-people.md]: urbit-dev is [here](https://groups.google.com/a/urbit.org/forum/#!forum/dev).

GitHub should link to just urbit/urbit.

File: content/understanding-urbit/history-people.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+Building Urbit is about building a future where technology doesn’t control us. Instead, we look forward to a future where technology is simpler, more reliable and less invasive. 
+
+Keep in touch while we make build this vision into reality by [joining our mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) or following us on [twitter/urbit](https://twitter.com/urbit) or [github/urbit](https://github.com/urbit). 
+
+If you’re somewhat technical and you’d like to help and contribute, [install urbit](/using/install) and consider [applying for a grant](https://grants.urbit.org/s).
+
+If you’re interested in homesteading a community when OS 1 comes out, [apply here](community/community-grants).
+
+Or if you’re curious to try hoon school, [let us know](/community/hoonschool).
```
Comment (matildepark) [inline content/understanding-urbit/history-people.md]: capitalise

File: content/understanding-urbit/history-people.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+2019
+Tlon spends the year stabilizing Arvo and building OS 1.
+ 
+![](https://media.urbit.org/site/understanding-urbit/your-last-computer/your-last-computer-waves%402x.png)
+
+Building Urbit is about building a future where technology doesn’t control us. Instead, we look forward to a future where technology is simpler, more reliable and less invasive. 
+
+Keep in touch while we make build this vision into reality by [joining our mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) or following us on [twitter/urbit](https://twitter.com/urbit) or [github/urbit](https://github.com/urbit). 
+
+If you’re somewhat technical and you’d like to help and contribute, [install urbit](/using/install) and consider [applying for a grant](https://grants.urbit.org/s).
```
Comment (matildepark) [inline content/understanding-urbit/history-people.md]: capitalise Urbit

File: content/understanding-urbit/history-people.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+urbit.live gets going and starts selling Urbit IDs to the public.
+
+2019
+Tlon spends the year stabilizing Arvo and building OS 1.
+ 
+![](https://media.urbit.org/site/understanding-urbit/your-last-computer/your-last-computer-waves%402x.png)
+
+Building Urbit is about building a future where technology doesn’t control us. Instead, we look forward to a future where technology is simpler, more reliable and less invasive. 
+
+Keep in touch while we make build this vision into reality by [joining our mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) or following us on [twitter/urbit](https://twitter.com/urbit) or [github/urbit](https://github.com/urbit).
```
Comment (matildepark) [inline content/understanding-urbit/history-people.md]: I'm unsure if "twitter/urbit" is the intentional link name.

**Comment by matildepark** [comment]
If we're introducing a new roadmap, then we should remove the link to ~2019.10's Roadmap on templates/index.html, too.

File: templates/understanding-urbit/list.html [hunk 8]
```diff
@@ -32,87 +32,79 @@
...
+  <img class="full mb4" src="https://storage.googleapis.com/media.urbit.org/site/understanding-urbit/intro/intro-0.jpg"/>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Urbit ID is an identity and authentication system specifically designed to work with Urbit OS. When you boot or log in to Urbit OS, you use your Urbit ID. </p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Your Urbit ID is a short, memorable name (like </code>~ravmel-ropdyl</code>) that’s a username, network address and crypto wallet all in one. It’s registered on a blockchain, you own it with a key and no one can take it away from you. Urbit IDs are cryptographic property. </p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Urbit IDs aren’t money, but they are scarce, so each one costs something. This means that when you meet a stranger on the Urbit network, they have some skin in the game and are less likely to be a bot or a spammer.</p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">The Urbit ID system is live and deployed, and we don’t plan to change it all that much. It works well. If you’re curious to dive further into the details, jump ahead to <a class="bb" href="/understanding-urbit/urbit-id">this post</a></p>
```
Comment (matildepark) [inline templates/understanding-urbit/list.html]: This needs a period at the end of the paragraph, it's dangling.

File: templates/understanding-urbit/list.html [hunk 9]
```diff
@@ -32,87 +32,79 @@
...
+          type="submit"
+          name="subscribe"
+          onclick="_paq.push(['trackEvent', 'Mailing List', 'Subscribe'])">
+          <span class="fr pr1">-></span>
+      </button>
+      </div>
+  </div>
+  </form>
+  <p class="full mt3 mb3 mb3-xl lh-copy">You can also follow us on <a class="bb" href="https://twitter.com/urbit">Twitter</a> or <a class="bb" href="https://github.com/urbit">Github</a>.</p>
+  <p class="full mt0 mb3 mb4-xl lh-copy">Or, preferably, on Urbit itself in <code>`~dopzod/urbit-help`</code></p>
```
Comment (matildepark) [inline templates/understanding-urbit/list.html]: If you're using a code element then the backticks should be removed.

File: templates/understanding-urbit/list.html [hunk 10]
```diff
@@ -32,87 +32,79 @@
...
-    We built Urbit so people can control their technology, shape their digital lives and freely build their digital communities.
-  </p>
-  <p class="full c4-10-lg lh-copy mt2">
-    We want to replace the Internet with a foundation for computing that we can all rely on far into the future.
-  </p>
-  <p class="full c4-10-lg lh-copy mt2">
-    Urbit is a large, ambitious project. But at the core it’s simple, durable and yours.
-  </p>
+<section class="c4-10-lg">
+  <h1 id="introduction" class="mb4 full mt4 ">What Urbit Is</h1>
```
Comment (matildepark) [inline templates/understanding-urbit/list.html]: Just personal taste, but "What Urbit Is" looks weird and almost internal on the actual deploy. "Introduction" is more natural, and all the links say that as well.

File: content/understanding-urbit/history-people.md [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+
+For those interested in contributing there are some simple instructions on [GitHub](https://github.com/urbit/urbit/blob/master/CONTRIBUTING.md#contributing-to-urbit), but we also run a [grants program](https://grants.urbit.org/) where we reward contributors with Urbit ID address space. Grants aren’t solely for pre-determined projects, either. If you have something you’d like to propose, we’d love to hear from you.
+
+We also run a bi-monthly course called ‘Hoon School’ to teach people about the system. Hoon School is designed for anyone with the most basic technical skills — you don’t have to be an expert programmer. If you’re interested, you should [apply](community/hoonschool). We accept everyone we can and only close the class when it gets too big.
+
+![](https://media.urbit.org/site/understanding-urbit/your-last-computer/your-last-computer-waves%402x.png)
+
+At the beginning, Urbit was just a few people with the right combination of imagination and discipline to try to rebuild computing.
+
+2002
```
Comment (matildepark) [inline content/understanding-urbit/history-people.md]: This timeline [doesn't have the line breaks](https://deploy-preview-329--urbit-org.netlify.com/understanding-urbit/history-people/) in the deploy. You'll need to add another line break.

File: content/understanding-urbit/history-people.md (hunk 7 above)
Comment (eriknewton) [reply content/understanding-urbit/history-people.md]: let's just call it "twitter"

File: templates/understanding-urbit/list.html (hunk 10 above)
Comment (eriknewton) [reply templates/understanding-urbit/list.html]: Agreed.

File: content/blog/simple-durable-yours.md (hunk 3 above)
Comment (urcades) [reply content/blog/simple-durable-yours.md:5]: Good catch

File: content/understanding-urbit/urbit-id.md (hunk 2 above)
Comment (urcades) [reply content/understanding-urbit/urbit-id.md]: I'll use what you've linked!

Comment (urcades) [reply content/understanding-urbit/urbit-id.md]: TYVM

File: content/understanding-urbit/history-people.md (hunk 4 above)
Comment (matildepark) [reply content/understanding-urbit/history-people.md]: The urbit-dev link is still pointing to Mailchimp, not to the Google group.

File: content/understanding-urbit/history-people.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+
+Given how new and unusual Urbit is, it attracts a wide variety of interesting people — both technical and non-technical. The community is generally happy to talk to people about what Urbit is, what we’re up to, and where we’re going. The best way to get involved is to simply join `~dopzod/urbit-help` — which our [setup instructions](/using/operations/using-your-ship/) can help you do.
+
+For those interested in contributing there are some simple instructions on [GitHub](https://github.com/urbit/urbit/blob/master/CONTRIBUTING.md#contributing-to-urbit), but we also run a [grants program](https://grants.urbit.org/) where we reward contributors with Urbit ID address space. Grants aren’t solely for pre-determined projects, either. If you have something you’d like to propose, we’d love to hear from you.
+
+We also run a bi-monthly course called ‘Hoon School’ to teach people about the system. Hoon School is designed for anyone with the most basic technical skills — you don’t have to be an expert programmer. If you’re interested, you should [apply](community/hoonschool). We accept everyone we can and only close the class when it gets too big.
```
Comment (matildepark) [inline content/understanding-urbit/history-people.md]: should be `/community/hoonschool`

File: content/understanding-urbit/history-people.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+ 
+![](https://media.urbit.org/site/understanding-urbit/uu-history-4.jpg)
+
+Building Urbit is about building a future where technology doesn’t control us. Instead, we look forward to a future where technology is simpler, more reliable and less invasive. 
+
+Keep in touch while we make build this vision into reality by [joining our mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) or following us on [Twitter](https://twitter.com/urbit) or [GitHub](https://github.com/urbit). 
+
+If you’re somewhat technical and you’d like to help and contribute, [install Urbit](/using/install) and consider [applying for a grant](https://grants.urbit.org/s).
+
+If you’re interested in homesteading a community when OS 1 comes out, [apply here](community/community-grants).
```
Comment (matildepark) [inline content/understanding-urbit/history-people.md]: should be `/community/community-grants`

File: content/understanding-urbit/roadmap.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+
+OS 1 is aimed first and foremost at communities looking to homestead a new platform they can truly make their own. We’ve been using Urbit ourselves over the years, and OS 1 is what we’ve been looking forward to using for our own daily communication and collaboration.
+
+While we’re excited about what OS 1 is, there are also plenty of gaps. For example, OS 1 doesn’t have an exceptional notification system, it still lives in a browser, and it lacks great tooling for distributing new modules. While we aren’t inventing any new cryptography, Urbit also hasn’t been audited, so we aren’t calling it secure just yet. 
+
+If OS 0 is for explorers, OS 1 is for tinkerers. More like car camping than backpacking.
+
+Tlon is also planning to provide free hosting to a limited number of groups who want to homestead the system. (Although you can always host Urbit yourself, of course.) If you’re interested in trying OS 1 with your group of friends, [let us know](community/community-grants).
```
Comment (matildepark) [inline content/understanding-urbit/roadmap.md]: `/community/community-grants`

File: templates/understanding-urbit/list.html [hunk 15]
```diff
@@ -32,87 +32,79 @@
...
+  <p class="full mt0 mb4 mb4-xl lh-copy">In an Urbit world, each person has their own Urbit OS node, or ‘urbit’. Your urbit is secure and private to you and entirely under your control. When you want to connect with others, you connect to their urbit directly — rather than going through a centralized service. </p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">If you’re curious to dive even deeper into Urbit OS, feel free to <a class="bb" href="/understanding-urbit/urbit-os">skip ahead</a>.</p>
+  <img class="full mb4" src="https://media.urbit.org/site/understanding-urbit/project-history/intro-id.svg"/>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Urbit ID is an identity and authentication system specifically designed to work with Urbit OS. When you boot or log in to Urbit OS, you use your Urbit ID. </p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Your Urbit ID is a short, memorable name (like </code>~ravmel-ropdyl</code>) that’s a username, network address and crypto wallet all in one. It’s registered on a blockchain, you own it with a key and no one can take it away from you. Urbit IDs are cryptographic property. </p>
```
Comment (matildepark) [inline templates/understanding-urbit/list.html]: `</code>~ravmel-ropdyl</code>` should be `<code>~ravmel-ropdyl</code>`

**Comment by galenwp** [comment]
The only things I see missing:

- Homepage content from the Figma mocks
- 1px borders on the images. I like how we were consistent about this before.

**Comment by matildepark** [comment]
I'm working on that feedback — I noticed Roadmap and OS1 -> N are the same post, but a lot of the content links to OS1 -> N, so I'm gonna rectify that and add an alias in the post, too.

**Comment by galenwp** [comment]
Good catch, thanks.

**Comment by matildepark** [comment]
Ready for another look @galenwp.

**Comment by galenwp** [comment]
Edited the copy, changed the homepage a bit, take a look

**Comment by matildepark** [comment]
image

I don't know about the italics and I really don't think we need to link to UU. It's directly below it.

**Comment by galenwp** [comment]
Reading that intro copy, I found it really confusing to see 'Install Urbit OS' above UU. For people who have never seen Urbit before, I want to make sure they head to UU before trying to install.

**Comment by matildepark** [comment]
There are very few situations in which a fresh user won't be able to see the words "Understanding Urbit", like, directly below that installation link, and it looks redundant and sort of odd (why is it green? What is that trying to signify?).

Some of our users are going to come knowing what Urbit is and wanting the install link. An increasing number, really — they have to come back for updated binaries. The original one had "Introduction" in black, install and docs below it for alternative streams into the site if you know what you want — very natural. 

And if we want people to return to this site — to tap our content streams — we need to stop so aggressively formulating an introductory funnel for everyday people to read it and bounce.

I'd even go so far as to say that "Introduction / Install / Documentation" link titles are optimal, cleanest.

But I know @eriknewton said this is iterative and meant to go out ASAP, so I can see very little room to argue much further here. You just said to take a look.

**Comment by galenwp** [comment]
Sending new visitors to 'install urbit' is going to be confusing. Anyway, this is a layout issue. I think those links are a relic and can go. Ed has a way to reorganize the text into a three-up that he's working on. I'll push something that should work for the time being.

**Pull request merged**